src/solver/rosenbrock.cu

```diff
@@ -263,7 +304,7 @@ namespace micm
         if (stat != CUBLAS_STATUS_SUCCESS)
         {
           printf(cublasGetStatusString(stat));
-          throw std::runtime_error("Error while calling cublasDnrm2.");
+          throw std::system_error(make_error_code(MicmCudaRosenbrockSolverErrc::CublasError), "");
```

Comment:
```suggestion
          throw std::system_error(make_error_code(MicmCudaRosenbrockSolverErrc::CublasError), "cublasStatus_t came back as " + stat);
```
Just to provide the actual error code to the user.

Reply:
if that's the case should the printf the line above be removed?

Reply:
I would say yes since I don't think we need to print to stdout anymore if there's an error since we are throwing the exception now.

Reply:
yeah, I agree - I try to remove output to stdout and stderr when I find them. This should be updated now.